NitroxServer/Serialization/World/WorldPersistence.cs

```diff
@@ -86,6 +88,69 @@ internal bool Save(PersistedWorldData persistedData, string saveDir)
             }
         }
 
+        public bool BackUp(string saveDir)
+        {
+            string backupDir = Path.Combine(saveDir, "Backups");
+            if (!Directory.Exists(backupDir))
+            {
+                Directory.CreateDirectory(backupDir);
+            }
+
+            // Backup up the save files
+            string outZip = Path.Combine(backupDir, $"Backup - {DateTime.Now:yyyyMMddHHmmss}");
+
+            try
+            {
+                Directory.CreateDirectory(outZip);
+
+                foreach (string file in Directory.GetFiles(saveDir))
+                {
+                    File.Copy(file, Path.Combine(outZip, Path.GetFileName(file)));
+                }
+
+                FileSystem.Instance.ZipFilesInDirectory(outZip, $"{outZip}.zip");
+                Directory.Delete(outZip, true);
+
+                // Check for total number of backups and prune as necessary
+                List<string> backups = new();
+                foreach (string file in Directory.EnumerateFiles(backupDir))
+                {
+                    FileInfo fileInfo = new(file);
+
+                    if (fileInfo.Extension != ".zip" || !fileInfo.Name.Contains($"Backup - "))
```

Comment:
We can definitely discuss that with the other devs in the developer channels of the Discord server (once you get the Jr-Alterra role).

I don't think allowing users to rename their backup files is necessary. Ideally, all they would have to do when their world corrupts is to backup up the most recent save file, and the extra backups are there in case some of the recently-made ones are also corrupted.